homeworkNew.md

```diff
@@ -0,0 +1,111 @@
+# Linus's Homework
+
+## Question 1
+
+What is 2 + 2?
+
+4
+
+## Question 2
+
+What is the opposite of Black?
+
+White
+
+## Question 3
+
+What is the  2 * 3 * 6 * 1 * 10 * 34?
+
+12240
+
+## Question 4 
+
+What does HTML stand for?
+
+Hyper Text Markup Language
+
+## Question 5
+
+What is CSS useful for?
+
+Making a website look better (style, size, position, color and so on)
+
+## Question 6
+
+Using CSS, how would I select every `<h1>` tag on the page?
+
+'''css
+#h1 {}
+```
+
+## Question 7
+
+What is JavaScript?
+
+A programming language used to create and control dynamic website content.
+
+## Question 8
+
+What three problem does Git & Github solve?
+
+It's a virsion control system, it lets you manage and keep track of your source code history. 
```

Comment:
Hi @IrinShilova 
Well done for your effort. You may want to look at the spelling of 'version' on line 51.

Reply:
Thank you, Amarachi